ui/eureka/eureka.js

```diff
@@ -173,6 +173,7 @@ const Options = {
 
     // Bozja
     // Southern Front: https://xivapi.com/search?indexes=Fate&filters=ID%3E=1597,ID%3C=1628&columns=Description,Name,Url
+    // Zadnor: https://xivapi.com/search?indexes=Fate&filters=ID%3E=1717,ID%3C=1742&columns=Description,Name,Url
```

Comment:
Right now I've added all the xivapi links in one block before each zone section. Would you prefer them inline with their respective zone? So it'd look like
```cs
    // Eureka
    // Anemos:  https://xivapi.com/search?indexes=Fate&filters=ID>=1328,ID<=1348&columns=Description,Name,Url
    [ZoneId.TheForbiddenLandEurekaAnemos]: {
      ...
    },
    // Pagos:   https://xivapi.com/search?indexes=Fate&filters=ID>=1351,ID<=1369&columns=Description,Name,Url
    [ZoneId.TheForbiddenLandEurekaPagos]: {
      ...
    }.
    ...

    ...
    // Bozja
    // Southern Front: https://xivapi.com/search?indexes=Fate&filters=ID%3E=1597,ID%3C=1628&columns=Description,Name,Url
    [ZoneId.TheBozjanSouthernFront]: {
      ...
    },
    // Zadnor: https://xivapi.com/search?indexes=Fate&filters=ID%3E=1717,ID%3C=1742&columns=Description,Name,Url
    [ZoneId.TheBozjanSouthernFront]: {
      ...
    },
```

Reply:
I think inline makes more sense to me, but I think either is fine for now as an initial PR. I was thinking also that it might make sense (in some followup cleanup) to break each of these zones up into their own file as well (which would make separate comments make sense too).

Reply:
Yeah, I had considered splitting it too, will do it in the next sweep of changes.